tool/minimap.re

```diff
@@ -144,6 +174,27 @@ fn create_minimap_menu() -> Ui {
                 }
             },
         }),
+        UiElement::Chooser(Chooser {
+            label: Text { text: "Position" },
+            options: List::of(
+                Text { text: "Top Left" },
+                Text { text: "Top Right" },
+                Text { text: "Bottom Left" },
+                Text { text: "Bottom Right" },
+            ),
+            selected: pos.number,
+            onchange: fn(index: int) {
+                match index {
+                    0 => { SETTINGS.minimap_position = "TopLeft"; },
+                    1 => { SETTINGS.minimap_position = "TopRight"; },
+                    2 => { SETTINGS.minimap_position = "BottomLeft"; },
+                    3 => { SETTINGS.minimap_position = "BottomRight"; },
+                    _ => { SETTINGS.minimap_position = "TopRight"; }
```

Comment:
The catch-all case should panic with an error, as it indicates a bug in our code. It shouldn't silently fail with garbage.